tests/eslint-plugin/rules/no-cycle.test.ts

```diff
@@ -2,7 +2,8 @@ import { run } from '../utils'
 
 function createCycleSourceError(p: string) {
   return {
-    message: `Dependency cycle via ${p}`,
+    messageId: 'cycleSource',
+    data: { source: p },
   }
 }
```

Comment:
`eslint-plugin-import-x@4.5.0` has removed all the `message` and has migrated them all to the `messgaeId`. So the error has also been updated.